isaric/parsers/ccp-guinea.toml

```diff
@@ -667,9 +675,11 @@
   ref = "Y/N/NK"
 
 [visit.treatment_noninvasive_ventilation]
-  field = "overall_noninvasive_proccur"
-  description = "Non-invasive ventilation (eg BiPAP, CPAP)?"
-  ref = "Y/N/NK"
+  combinedType = "any"
+  fields = [
+    { field = "overall_noninvasive_proccur", description = "Non-invasive ventilation (eg BiPAP, CPAP)?", ref = "Y/N/NK" },
+    { field = "overall_oxygen_cmtype___5", description = "CPAP/NIV mask", values = { 2 = true } },
+  ]
```

Comment:
@pipliggins looking back at my initial review, seems there is no data collected in many of these overall_ variables. the variable non_inv_vent can be added, but data is only available for a small number of ICU patients